python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -87,24 +87,27 @@ def get_runs(graphene_info, filters, cursor=None, limit=None):
     check.opt_int_param(limit, "limit")
 
     instance = graphene_info.context.instance
-    runs = []
 
     if filters and filters.run_ids and len(filters.run_ids) == 1:
         run = instance.get_run_by_id(filters.run_ids[0])
         if run:
-            runs = [run]
-    elif filters and (
+            return [GrapheneRun(run)]
+        else:
+            return []
```

Comment:
should we still do this "shortcut" path? seems bad now that we want to root on `record`

Reply:
yeahhhhhhh.... I should do this, I was just being lazy.